### ISSUE TITLE
Bug: WinUI-2.5: Accessibility: [MAS4.2.1]: The screen reader user is not aware about the availability of the spinner when user is navigating with the Tab key

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#3615]
**Tags**
Accessibility, A11yFT,  A11yMAS, A11yMediumImpact, MAS4.2.1, Narrator, Sev3, WinUI-2.5, Win32, Wipro 

**User Impact**
The screen reader user is not aware about the availability of the spinner when user is navigating with the Tab key. So the screen reader user might face difficulty to navigate/find it.

**Steps to reproduce the bug**
Launch the WinUI_2.5_TestApp
1. Go to 'Progress Spinner'
2. Run the Accessibility Insight for Windows for the Progress spinner.
3. Observe the "Is Keyboard Focusable" property for the spinner. 

**Expected behavior**
1. The screen reader user should be able to know about the availability of the spinner using Tab key also.
2. "Is keyboard Focusable" should be set as "true" for the Progress Spinner element.

**Actual behavior**
1. The screen reader user is not aware about the availability of the spinner when user is navigating with the Tab key. Although using Caps + Left/Right Key user is able to traverse the spinner element.
2. "Is keyboard Focusable" is false instead of true for the Progress Spinner element.

**Screenshots**
![image](https://user-images.githubusercontent.com/74585893/99381120-d83bec80-28f0-11eb-9ec4-65ed6e0daf82.png)

**Version Info**
Win32
Windows OS Version: Windows 10
App: WinUI_2.5_TestApp

**Comment by ranjeshj** [comment]
@SavoySchuler Any particular reason why NumberBox is not a tab stop by default ?

**Comment by chingucoding** [comment]
I think it should be focusable or if not focusable, should not expose itsself as spinner. @YuliKl FYI
